TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -227,18 +227,22 @@ class TestNSHTTPCookieStorage: XCTestCase {
         storage.setCookie(testCookie)
         XCTAssertEqual(storage.cookies!.count, 3)
         var destPath: String
+        //let bundleName = Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
+        //TODO: Above api returns nil on Linux when this code is introduced. Hence, using the below workaround.
```

Comment:
Sounds like an expected failure. A macOS command line tool won't necessarily have an Info.plist with the bundle display name either. I don't think you should check in this commented-out code.